tests/validate.test.js

```diff
@@ -518,6 +518,21 @@ describe('validate', () => {
               }
             }
           ]
+        },
+        dockerfuncuri: {
+          image: {
+            name: 'some-image-with-uri',
+            uri: 'http://hub.dock.er/image',
+            command: ['method.lambda']
+          },
+          events: [
+            {
+              http: {
+                method: 'POST',
+                path: 'mydockerfuncpath'
+              }
+            }
+          ]
         }
```

Comment:
This does not seem to test that URIs are skipped.

Reply:
I've added a dedicated test!